chains/edgenchain/metadata.yaml

```
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiKey: testkey
+    apiUrl: https://edgenscan.io/api
+    family: blockscout
+    name: Edgenscan
+    url: https://edgenscan.io
+chainId: 4207
```

include `blocks` config e.g.
```suggestion
blocks:
  confirmations: 1
  estimateBlockTime: 3
  reorgPeriod: 5
chainId: 4207
```